### PR TITLE
b and c integrations: minor style change

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

b and c integrations: minor style change
- apply sentence-style capitalization
- add some references to glossary terms

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/binary_sensor.rest.markdown [hunk 1]
```diff
@@ -84,7 +84,7 @@ name:
   description: Name of the REST binary sensor.
   required: false
   type: template
-  default: REST Binary Sensor
+  default: REST binary sensor
```
Comment (ludeeus) [inline source/_integrations/binary_sensor.rest.markdown]:
```suggestion
  default: REST Binary Sensor
```

This describes the default value set by the integration, so unless the integration has been changed in core, this should not change.

File: source/_integrations/bosch_shc.markdown [hunk 2]
```diff
@@ -49,7 +49,7 @@ The cover platform allows you to control your covers. Cover devices are added fo
 
 The sensor platform allows you to monitor the states of your temperature, humidity, purity, air quality, power, energy, and valve tappet sensors. Sensor devices are added for each of the following devices:
 
-- Thermostat
+- thermostat
```
Comment (ludeeus) [inline source/_integrations/bosch_shc.markdown]:
```suggestion
- Thermostat
```

File: source/_integrations/broadlink.markdown [hunk 3]
```diff
@@ -552,7 +552,7 @@ First get or learn all the remotes you want to add to Home Assistant in e-Contro
 
 This is the code we need to transmit again to replicate the same remote function.
 
-### Using Node red to Transmit Codes
+### Using Node Red to transmit codes
```
Comment (ludeeus) [inline source/_integrations/broadlink.markdown]:
```suggestion
### Using Node-RED to transmit codes
```
<https://nodered.org/> that's how they write it.

File: source/_integrations/cloudflare.markdown [hunk 4]
```diff
@@ -31,7 +31,7 @@ An easy way to create this is to start with the "Edit zone DNS" template then ad
 
 ### Usage of external service
 
-This integration uses the whoami service from [home-assistant/services.home-assistant.io](https://github.com/home-assistant/services.home-assistant.io) to set the public IP address.
+This integration uses the whoami {% term service %} from [home-assistant/services.home-assistant.io](https://github.com/home-assistant/services.home-assistant.io) to set the public IP address.
```
Comment (ludeeus) [inline source/_integrations/cloudflare.markdown]:
```suggestion
This integration uses the whoami service from [home-assistant/services.home-assistant.io](https://github.com/home-assistant/services.home-assistant.io) to set the public IP address.
```

This is not a service inside Home Assistant but a public web service.

File: source/_integrations/compensation.markdown [hunk 5]
```diff
@@ -14,7 +14,7 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Compensation {% term integration %} consumes the {% term state %} from other {% term sensors %}. It exports the compensated value as state in a separate {% term entity %} and the following values as attributes: `entity_id` and `coefficients`.  A single polynomial, linear by default, is fit to all data points provided.
+The **Compensation** {% term integration %} consumes the {% term state %} from other {% term sensors %}. It exports the compensated value as state in a separate {% term entity %} and the following values as attributes: `entity_id` and `coefficients`.  A single polynomial, linear by default, is fit to all data points provided.
```
Comment (ludeeus) [inline source/_integrations/compensation.markdown]:
```suggestion
The **Compensation** {% term integration %} consumes the {% term state %} from other {% term sensors %}. It exports the compensated value as state in a separate {% term entity %} and the following values as attributes: `entity_id` and `coefficients`. A single polynomial, linear by default, is fit to all data points provided.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**